Classes/Parser/ScssParser.php

```diff
@@ -83,10 +84,36 @@ protected function parseFile($file, $settings)
                 'sourceMapBasepath' => '<PATH DOES NOT EXIST BUT SUPRESSES WARNINGS>'
             ]);
         }
-        $css = $scss->compile('@import "' . $file . '"');
 
         $absoluteFilename = GeneralUtility::getFileAbsFileName($file);
-        $relativePath = $settings['cache']['tempDirectoryRelativeToRoot'] . dirname(substr($absoluteFilename, strlen($this->getPathSite()))) . '/';
+
+        // Make paths in url() statements relative to site root
+        $absoluteFilePath = dirname($absoluteFilename);
+        $siteFilePath = dirname(substr($absoluteFilename, strlen($this->getPathSite())));
+        $absoluteBootstrapPackagePath = GeneralUtility::getFileAbsFileName(ExtensionManagementUtility::extPath('bootstrap_package') . 'Resources/Public/Scss/Theme');
+        $siteBootstrapPackagePath = substr($absoluteBootstrapPackagePath, strlen($this->getPathSite()));
```

Comment:
I'd name the variable to something that represents what is meant, e.g. `$bootstrapPackageThemeWebPath` and use the API method (see above) to derive it from absolute path